modules/web/src/com/haulmont/sampler/web/ui/components/icons/simple/simple-icons.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<window xmlns="http://schemas.haulmont.com/cuba/window.xsd"
+        messagesPack="com.haulmont.sampler.web.ui.components.icons.simple"
+        class="com.haulmont.sampler.web.ui.components.icons.simple.SimpleIconsFrame">
+    <layout>
+        <hbox spacing="true">
+            <groupBox id="groupFont"
+                      caption="msg://caption.groupFont"
+                      spacing="true">
+                <label icon="EUR"
+                       value="EURO"/>
+                <label icon="font-icon:BITCOIN"
+                       value="BITCOIN"/>
+            </groupBox>
+            <groupBox id="groupSimple"
+                      caption="msg://caption.groupSimple"
```

Comment:
To be honest, I don't like this "simple" name. I suggest grouping icons by declarative and programmatic definition.